packages/cli/src/utils/flags.ts

```diff
@@ -33,9 +33,21 @@ export const ViewCommmandFlags = {
 export const MultiSigFlags = {
   useMultiSig: Flags.boolean({
     description: 'True means the request will be sent through multisig.',
+    exclusive: ['useSafe'],
   }),
   for: CustomFlags.address({
     dependsOn: ['useMultiSig'],
     description: 'Address of the multi-sig contract',
   }),
 }
+
+export const SafeFlags = {
+  useSafe: Flags.boolean({
+    description: 'True means the request will be sent through safe.',
```

Comment:
uppercase safe => SAFE. 

reason: because the name is so generic it sounds like we are just saying it will be sent safely. and SAFE is how its branded.

Reply:
Sounds good